### PR TITLE
Update Sentinel docs

### PR DESCRIPTION
**Pull request by jmcgill298**
Update Sentinel docs to have 3 hosts (minimum per Redis docs), and change `CELERY_BROKER_URL` to a multiline string instead of a Tuple (tuple is invalid, and raises an exception when job completes).

### CODE REVIEW

File: nautobot/docs/additional-features/caching.md [hunk 1]
```diff
@@ -138,7 +139,8 @@ CACHEOPS_SENTINEL = {
     "db": 0,
     "locations": [
         ("mysentinel.redis.example.com", 26379),
```
Comment (nniehoff) [inline nautobot/docs/additional-features/caching.md:141]: it is possible to use a DNS entry here that resolves to the 3 sentinel servers as well

File: nautobot/docs/additional-features/caching.md [hunk 2]
```diff
@@ -163,8 +165,9 @@ redis_password = ""
 sentinel_password = ""
 
 CELERY_BROKER_URL = (
-    f"sentinel://:{redis_password}@mysentinel.redis.example.com:26379,",
-    f"sentinel://:{redis_password}@othersentinel.redis.example.com:26379",
+    f"sentinel://:{redis_password}@mysentinel.redis.example.com:26379;"
+    f"sentinel://:{redis_password}@othersentinel.redis.example.com:26379;"
+    f"sentinel://:{redis_password}@thirdsentinel.redis.example.com:26379"
```
Comment (lampwins) [inline nautobot/docs/additional-features/caching.md:170]: Should note that the final entry should *not* end with a semicolon. When it is present, Celery tries to append a localhost entry to the list.

Comment (jmcgill298) [reply nautobot/docs/additional-features/caching.md:170]: Per the docs, I did retry using a list and removing the trailing delimiter, but this did not work (as experienced previously).

https://docs.celeryproject.org/en/stable/userguide/configuration.html#broker-url

Comment (jmcgill298) [inline nautobot/docs/additional-features/caching.md:170]:
```suggestion
    f"sentinel://:{redis_password}@thirdsentinel.redis.example.com:26379"  # The final entry must not have the `;` delimiter
```

**Review by glennmatthews — APPROVED** [review]
Looks reasonable to me. Thanks!

File: nautobot/docs/additional-features/caching.md (hunk 2 above)
Comment (lampwins) [reply nautobot/docs/additional-features/caching.md:171]: Yeah in some cases (not sure of the specifics) the empty string is interpreted as localhost. This threw me for a loop recently. Not sure on the list, seems like their docs are off...

**Pull request merged**